src/io/BulkData.ts

```diff
@@ -38,16 +38,19 @@ const importAll = async function(file): Promise<void> {
   console.info('Loading import data...')
   const promises = arr.map(o => writeFile(o.filename, o.data))
   await Promise.all(promises)
+  await store.dispatch('cloudSync', { callback: null, condition: 'bulkDelete' }).catch(e => console.error(e))
   console.info('Import data loaded! Running startup...')
-  await Startup(Vue.prototype.$appVersion, Vue.prototype.$lancerVersion, store)
+  await Startup(Vue.prototype.$appVersion, Vue.prototype.$lancerVersion, store, true)
 }
 
 const clearAllData = async function(): Promise<void> {
   console.info('Erasing all COMP/CON data...')
   const promises = files.map(file => writeFile(file, ''))
-  await promises
+  await Promise.all(promises)
+  await store.dispatch('cloudSync', { callback: null, condition: 'bulkDelete' }).catch(e => console.error(e))
+  
   console.info('All data erased! Running startup...')
-  await Startup(Vue.prototype.$appVersion, Vue.prototype.$lancerVersion, store)
+  await Startup(Vue.prototype.$appVersion, Vue.prototype.$lancerVersion, store, true)
```

Comment:
Assume that if we wanted to delete or overwrite all of our local data, we don't want to re-pull that data from the cloud.  Flag the Cloud Sync as `bulkDelete` so the deletion can be toggled as the user desires.